cli/replicaset/roles.go

```diff
@@ -5,18 +5,57 @@ import (
 	"slices"
 )
 
-// ChangeRoleFunc is a function type for addition or removing a role.
+// RoleAction is a type that describes an action that will
+// happen with Roles. It can be "add" or "remove".
+type RoleAction uint
+
+const (
+	AddAction RoleAction = iota
+	RemoveAction
+)
+
+// ChangeRoleFunc is a function for adding/removing roles.
 type ChangeRoleFunc func([]string, string) ([]string, error)
 
-// AddRole is a function that implements addition of role.
+// RolesChangerAction implements changing (add or remove) of roles.
+type RolesChangerAction interface {
```

Comment:
```suggestion
// RolesChanger implements changing (add or remove) of roles.
type RolesChanger interface {
```

Reply:
We already have an interface with the same name.

Reply:
It looks like we have different levels of abstraction mixed in one place. It might be confusing, but let it be.

Reply:
Ok, then I'll just leave it at that.